recipes/sextractor/meta.yaml

```diff
@@ -37,8 +37,10 @@ test:
   source_files:
     - tests
   commands:
-    - cd tests
-    - s-extractor galaxies.fits -WEIGHT_IMAGE galaxies.weight.fits -CATALOG_NAME galaxies.cat
+    # These are simple existence tests.
+    # The actual test suite is run via autotools in build.sh.
+    - test -f ${CONDA_PREFIX}/bin/source-extractor
+    - test $(readlink -f ${CONDA_PREFIX}/bin/sex) = ${CONDA_PREFIX}/bin/source-extractor  # [linux]
```

Comment:
We will need to double-check with numfocus on this? @scopatz @mwcraig

Reply:
I'm fine with double-checking, but not having the symlink is a no-no for me. I need backwards compatibility.

Reply:
I figured once the people commenting here are done we would run this by numfocus again.

Reply:
I honestly don't like this, but I understand why people would want it. However, it seems easy for users to make whatever symlinks they like downstream of the actual package. So in light of that, i'd vote to not include it at all.

Reply:
To be a little more explicit, if this shortcut is *not* included the package will generate mostly complaints and frustration because all documentation for SExtractor uses the command line `sex`. 

Without the shortcut, users will install this, try running `sex`, figure they broke something, reinstall, etc, and eventually ask for help.

Reply:
I realize this, but I don't find that convincing personally.

Reply:
I'm a big +1 on including this symlink for backwards compatibility. A huge part of the reason we're doing this is to improve community dynamics. Breaking backwards compatibility incurs unnecessary friction for what is otherwise a purely additive, positive gain, and also potentially contributes to grumblings & negative sentiment about CoC and DISC matters.

Not breaking people's workflow - especially, in this case, a non-developer-facing command line tool - is really, really, really important.  (For instance, it's one of Linux Torvald's core principles is "we don't break userspace".)

Reply:
I think it is fine to have a backwards compatible link for the time being.